OpenRA.Mods.Common/Activities/Demolish.cs

```diff
@@ -57,6 +58,14 @@ protected override void OnInside(Actor self)
 				if (cloak != null && cloak.Info.UncloakOn.HasFlag(UncloakType.Demolish))
 					cloak.Uncloak();
 
+				if (!chargePlanted)
+				{
+					chargePlanted = true;
```

Comment:
Sorry I thought this runs on every tick like external capturing which I took as template. I will fix that.